### DISCUSSION TITLE
last version not working in kali

### DISCUSSION THREAD

**Discussion by webislands** [sherlock-project/sherlock#1453] [General]
i am install via git and pip requirement in last version of . run sherlock version and help is show launch command for search and wait no output just black wait 1h not show noting.

**Comment by Dargar30** [comment]
how to install kali on poco x3 pro by termux?

**Comment by Dargar30** [comment]
how to install kali on poco x3 pro by termux, step by step?

**Comment by Dargar30** [comment]
how to install kali on poco x3 pro by termux, step by step?
